[fix] failed to run make test in wsl2

**Summary**

failed to run the make test in wsl2
Description for this bug.

**How to reproduce**
run make test in wsl2

**Error message and/or stacktrace**

ulimit -c 1024 bash ./scripts/ci/ci-run-unit-tests.sh
/bin/sh: 1: ulimit: too many arguments
make: *** [Makefile:64：unit-test]

If applicable, add screenshots to help explain your problem.

**Additional context**

Add any other context about the problem here.

--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@ cli-test:
 	bash ./scripts/ci/ci-run-cli-unit-tests.sh
 
 unit-test:
-	ulimit -c 1024 bash ./scripts/ci/ci-run-unit-tests.sh
+	ulimit -n 10000; bash ./scripts/ci/ci-run-unit-tests.sh
 
 embedded-meta-test: build-debug
 	rm -rf ./_meta_embedded
@@ -69,7 +69,7 @@ embedded-meta-test: build-debug
 
 stateless-test: build-debug
 	rm -rf ./_meta/
-	ulimit -c 1024 bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
+	ulimit -n 10000; bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
 
 stateless-cluster-test: build-debug
 	rm -rf ./_meta/

